vpr/src/place/place_constraints.h

```diff
@@ -34,9 +34,19 @@ bool cluster_floorplanning_legal(ClusterBlockId blk_id, const t_pl_loc& loc);
 bool is_macro_constrained(const t_pl_macro& pl_macro);
 
 /*
- * Returns region of valid locations for the head of the macro based on floorplan constraints
+ * Returns PartitionRegion for the head of the macro based on the floorplan constraints
+ * of all blocks in the macro.
  */
```

Comment:
Shouldn't just do the head -- should figure out the tightest constraint on each member of the macro to give the most info possible to later code.